utils.py

```diff
@@ -0,0 +1,67 @@
+import requests # pylint: disable=import-error
+from google.cloud import language # pylint: disable=import-error
+from google.cloud.language import enums, types # pylint: disable=import-error
+
+SPANISH_API_URL = "http://sesat.fdi.ucm.es:8080/servicios/rest/sinonimos/json/"
+KEY_PARTS_OF_SPEECH = ["ADJ", "NOUN", "NUM", "VERB"]
+
+def gcloud_syntax_extraction(text):
+    """
+    Calls the gcloud natural langauge api's analyze syntax
+    Attributes:
+        text: the text which we wish to analyze
+    Returns:
+        the response given by the api call
+    """
+    client = language.LanguageServiceClient()
+    # pylint: disable=no-member
+    document = types.Document(
+        content=text,
+        type=enums.Document.Type.PLAIN_TEXT)
+    response = client.analyze_syntax(document=document)
+
+    return response
+
+def from_gcloud_remove_extra(gcloud_response):
```

Comment:
I would flip the functionality of `from_gcloud_remove_extra` and `gcloud_syntax_extraction` too.

The high-level goal here is really just a function that takes in a query and returns keywords, so your function really probably look something like `extract_keywords(query)`, and behind-the-scenes that function calls `gcloud_syntax_extraction` and does some parsing of the response.

But the fact that you use the gcloud syntax extraction & remove certain parts of the response is an implementation detail, and something the caller of the function doesn't need to know about, so you don't want to expose it in the function stub.

It also makes it easier to switch out your implementation later, because if you choose to use AWS syntax extractin or something like that, anyone calling `extract_keywords(query)` won't break, but anyone using `utils.from_gcloud_remove_extra(utils.gcloud_syntax_extraction(query))` will.